content/about/security-release-process/security-release-process.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Security Release Process
+description: Release process for Istio security vulnerabilities.
+weight: 35
+page_icon: /img/security-release.svg
+---
+
+The Istio team follows the process below when releasing patches for critical
+security vulnerabilities.
+
+## Security vulnerability disclosures
+
+Please refer to the [disclosure process](/about/security-vulnerabilities/).
+
+## Istio security team
+
+Security vulnerabilities should be handled quickly and privately.
+The primary goal of this process is to reduce the total time customers are
+vulnerable to publicly known exploits.
+
+The Istio security team is responsible for organizing the entire response
+including internal communication and external disclosure. The team enlists help
+from relevant developers and release leads to successfully run this process.
+
+The Istio security team consists of volunteers subscribed to the
+private [`istio-security-vulnerabilities@google.com`](mailto:istio-security-vulnerabilities@google.com)
+list.
+
+### Istio security team membership
+
+Membership to the Istio security team is based on the roles defined below.
+
+#### Roles
+
+* Fix lead
+
+Per issue, Fix Lead sees the issue through to the end.
+
+* Release lead
+
+Per issue, Release Lead is in charge of releasing the fix.
+
+* Disclosure lead
+
+Handles public messaging around the bug, and the documentation of how to upgrade.
+Explains the severity of the vulnerability and requests CVEs (Common Vulnerability Exposures).
```

Comment:
Avoid using tacit subjects. English grammar requires a subject for all sentences and the repetition helps the reader learn the terminology you are introducing:
* **Disclosure lead:** The disclosure lead handles the public messaging around the bug documents how to upgrade. Additionally, this role explains the severity of the vulnerability and requests Common Vulnerability Exposures (CVEs).